webrtc.html

```diff
@@ -1520,13 +1523,14 @@ <h4>Set the RTCSessionDescription</h4>
                             to <code>"closed"</code>.</p>
                           </li>
                           <li>
-                            <p>Fire an event named <code><a data-link-for=
-                            "RTCDataChannel">error</a></code > with an
+                            <p><a>Fire an event</a> named <code><a data-link-for=
+                            "RTCDataChannel">error</a></code > using the
+                            <code><a>RTCError</a></code> interface with an
                             <code>OperationError</code> exception at
                             <var>channel</var>.</p>
```

Comment:
This was already weird I suppose, but all events are supposed to use an interface that inherits from `Event`. I suspect this won't work if you actually follow all the steps "fire an event" lays out.